build.sh

```diff
@@ -13,7 +13,7 @@ java_scripts_folder="${script_location}/resource/java_scripts"
 bin_folder="${script_location}/bin"
 cStringifyFile_script="${script_location}/resource/script/cStringifyFile.pl"
 qos_file="${script_location}/perftest_qos_profiles.xml"
-
+doc_folder="${script_location}/srcDoc"
```

Comment:
this is for the sources, but the generated documentation should be just in ./doc, not inside this folder.

Reply:
done